<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LicenseDetailsDialog</class>
 <widget class="QDialog" name="LicenseDetailsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>319</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>License Details</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Generic</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>License Type:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="licenseTypeLabel"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>License Key:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="licenseKeyLabel"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Locked to Hardware ID:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="licenseHwidLabel"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="featuresGroupBox">
        <property name="title">
         <string>Features</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QFormLayout" name="featuresLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="errorLabel">
        <property name="text">
         <string notr="true">&lt;This license type requires higher software version.&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
